### DISCUSSION TITLE
Akash: 2025 Events Proposal

### DISCUSSION THREAD

**Discussion by vakaytion** [akash-network/community#748] [Governance Proposals]
As 2024 comes to a close, the Akash core team is happy to share a look ahead at the events plan for 2025. Akash’s brand is strong, and the evidence was very clear as our team took part in events around the world throughout 2024. The Akash core team and Insiders attended events on five continents and with a total of 11 major blockchain and AI events along with additional one-day events, hackathons, and summits. 

One of the goals for events in 2024 was to attend a wide range of events to determine which ones would be valuable for future years. The outcome of what we found from 2024 events has a big impact on what we plan to incorporate into the updated event budget tiers going forward into 2025.

## 2025 Event Budget Tier Breakdown

For 2025, the goal is to achieve a 50/50 ratio of AI to Blockchain events. With recent advancements in AI, AI agents, and model training, we want to begin to build brand awareness in some of the big AI and machine learning conferences in addition to the crypto and blockchain events.

The event budget tiers below reflect the updated strategy for 2025, which includes event sponsorship, travel, and costs of attending. Tier 1 is the largest adjustment for the 2025 budget. We are choosing to push this tier budget significantly to maximize our brand, presence, and output at the events. This will give Akash more opportunities to speak in front of larger audiences, larger booth sizes (for greater engagement with attendees), along with hosting larger Akash-hosted events to complement the main conferences. 

### Tier 1: $287,576

This is the largest tier for Akash events. A Tier 1 event will have the following:

* Budget for up to 7 Overclock Labs and Akash community members to attend.   
* Larger overall event sponsorship. Including: multiple speaking opportunities and a 10’ x 20’ booth.   
* Larger budget for Akash side events to accommodate up to 300 guests.   
* Larger budget for marketing collateral (including signage, merchandise, booth decorations, and more)  
* A total of four Tier 1 events are planned for 2025 (EthDenver, NVIDIA GTC, Token2049 Singapore, and NeurIPS).

### Tier 2: $86,765

This tier represents the middle tier for Akash events. A Tier 2 event will have the following: 

* Budget for up to 5 Overclock Labs and Akash community members to attend.   
* Event sponsorship must include a speaking opportunity, with the option to include a booth (dependent on the remaining budget, to be determined on a case-by-case basis).  
* A budget for Akash-hosted side events to accommodate up to 150 guests.

### Tier 3: $32,492

This tier represents the smallest tier for Akash events. A Tier 3 event will have the following: 

* Budget for up to 3 Overclock Labs and Akash community members to attend.  
* Overall event sponsorship is smaller than a Tier 2 event with no formal event sponsorship or booth.   
* Budget to co-sponsor an integration partner’s events, or other events that align with the overall vision of Akash.  
* The team will be attending the main event, aligned side events, and tapping into potential speaking and networking opportunities. 

### Overage Budget

This allotment of money will be used at the discretion of the Overclock Labs team and community for attending events that fall into the categories below:

* Hackathons   
* Single-day events that are aligned with Akash  
* University Blockchain Club events

### 2025 Total Events Budget Request

Q1: $575,152  
Q2: $97,476  
Q3: $151,750  
Q4: $835,450  
Overage: $150,000   
**2025 Total Budget: $1,809,829**

## Events Attendance Schedule

This proposal covers attendance of events below which correlated tiers for each event.

| Quarter | Event Name | Tier | Budget Amount  |
| :---- | :---- | :---- | :---- |
| Q1 | EthDenver | 1 | $287,576 |
| Q1 | NVIDIA GTC | 1 | $287,576 |
| Q2 | Token2049 Dubai | 3 | $32,493 |
| Q2 | SuperAI Singapore | 3 | $32,493 |
| Q2 | Permissionless | 3 | $32,493 |
| Q3 | ICML Vancouver | 2 | $86,766 |
| Q3 | ETH CC | 3 | $32,493 |
| Q3 | Korea Blockchain Week | 3 | $32,493 |
| Q4 | KubeCon | 2 | $86,766 |
| Q4 | Messari | 2 | $86,766 |
| Q4 | Cosmoverse | 2 | $86,766 |
| Q4 | Token2049 Singapore | 1 | $287,576 |
| Q4 | NeurIPS Vancouver | 1 | $287,576 |

## Operational Notes

The 2025 Events budget will be managed by the Overclock Labs team: Zach Horn (Head of Marketing), Amanda Keay (Events Manager), and Scott Hewitson (Director of Finance), along with support from the Akash community.

Funds will be held in a new, separate wallet so that funds from any other source are not commingled. 

## Next Steps

The Akash community met during a working group session to discuss the plan above and incorporate feedback. Please find the recording of the working group meeting along with a transcript in the comments below.

**Comment by PhoeTheNix** [comment]
I have post the Hongkong Web3 Festival proposal a few weeks ago, and looks like it is totally missed, I respect the decision making of the team and Hongkong Web3 Festival is always welcome the joining of Akash community, thanks a lot.

**Reply by vakaytion** [reply]
Thank you! We will take a look at that conference as a potential moving forward

**Comment by brewsterdrinkwater** [comment]
As mentioned above, there was a working group session on November 20th, 2024. There is a working group channel in the Akash Network Discord called (wg-events-2025).

Please find the notes, transcript, and recording link here: https://github.com/akash-network/community/blob/main/wg-events/meeting-2025/wg-events-001-2024-11-20.md

**Comment by ghost** [comment]
Thank you for putting up for discussion, I believe Akash team did an outstanding job throughout 2024 and I feel the wider crypto community appreciates that. The team is pushing to do a bigger job this year, using basically the same budget. I will support and hope for a successful 2025

**Reply by vakaytion** [reply]
thank you! @instafinanzas great to have you in attendance at Cosmoverse this year!

**Comment by piber-dev** [comment]
great proposal! just like the previous year, i'm in favor of it. fun to see new strategies being utilized in how the funding should be prioritized differently at different events. 

one question though, for e.g. tier 3 events the budget is for "up to 3 Overclock Labs and Akash community members to attend," is there plan to add ranges in what the budget allows? such as, 3 international guests or 6 more local guests? 

i know that the google sheet has strict budgets for maximum flight price, maximum hotel price, etc, but it would make sense to have different budget costs for different guests to allow for a larger range in attendance. for example, a round-trip flight can cost less than $500 for someone to europe to travel to istanbul, but likely more than $1000 for many NA countries.

perhaps this is too hard to count for when flights and hotels can already skyrocket during the dates of a large event, but would love to see something like this in this proposal or a future one!

**Reply by vakaytion** [reply]
Thanks Piber! 

Yes, our estimates on flights, hotels etc take into account realistic expectations of high traffic areas and general flight prices. 

Of course, in country flights are cheaper but we have a max budget in place as to not abuse flight dollars. 

Thanks for the comment

**Comment by v0nerd** [comment]
I appreciate the comprehensive overview provided in the 2025 Events Proposal. The proposal sets a solid foundation for 2025. By incorporating these suggestions, we can enhance our impact and further strengthen the Akash brand in the blockchain and AI communities. Here are my thoughts and suggestions to enhance our strategy:

This proposal is much stronger than others:
Diverse Event Selection: The focus on a balanced mix of AI and blockchain events is commendable. This approach will maximize our brand exposure across different sectors.
Clear Budget Tiers: The tiered budget breakdown offers transparency and clarity, making it easier to understand resource allocation and expectations for each event.

And here are my insights into some things to be moderated.

Marketing Collateral:
Ensure that marketing materials are not only visually appealing but also effectively communicate our value proposition. Consider creating tailored content for different audiences at each event.

Leverage Virtual Platforms:

In addition to physical events, explore opportunities for virtual participation. This can help us reach a wider audience and engage with those who may not attend in person.

Thank you for considering my feedback. I’m excited about the upcoming events and look forward to contributing to our success!

**Reply by vakaytion** [reply]
thank you for your feedback and encouragement. 
I am 100% aligned and we aim to do everything professionally with crisp and appealing branding. We are tailoring our branding and marketing to fit either AI or Blockchain events. 

Regarding virtual platforms: we plan to continue to explore virtual participation such as hackathons, webinars, etc. We will not forget that amongst our physical travel. 

Thank you for your comments.

**Comment by CaptainSplince** [comment]
Great work @vakaytion - you have my vote!

**Comment by rodri-r** [comment]
Great proposal @vakaytion as you state "Akash’s brand is strong, and the evidence was very clear as our team took part in events around the world throughout 2024."

This proposal will surely help Akash's brand to become even stronger in 2025!

**Comment by cloud-j-luna** [comment]
I'm aligned with this proposal and very happy to see such a  wide range of events across several key ecosystems. Great work @vakaytion on 2024 and 2025 looking even brighter (and stylish on crocs 😆)

**Reply by vakaytion** [reply]
thank you luna! happy to hear it! Glad to have your support and hope to see you at some of these events. 
Crocs will definitely be sported

**Comment by vakaytion** [comment]
Since the original GitHub discussion in December, there have been some slight changes to the line items and final figures. The most significant adjustment is that the merchandise amount for Tier 1 events has increased from $10K to $20K. Additionally, the tax amount has been removed and will be recalculated and adjusted at the end of the year.

**Comment by SophoclesSummer** [comment]
This proposal was approved on 1/22/2025 by a 95% majority.
https://www.mintscan.io/akash/proposals/279

**Comment by vakaytion** [comment]
I would like to bring to your attention a minor adjustment to some events scheduled for quarters 2 and 3. 
Originally, we had planned for Akash to attend five Tier-3 events, including ETH CC, Messari Mainnet, Token2049 Dubai, Super AI in Singapore, and Kube Con. However, these events have been removed, and the allocated funds will now be redirected to support attendance at additional AI Summits, Hackathons, and larger AI and Machine Learning conferences. 
This was discussed on last month's Event Working Group call on January 29, 2025

**Comment by SophoclesSummer** [comment]
This proposal passed on chain (most recently) on August 20, 2025 by a 93% majority. 
https://www.mintscan.io/akash/proposals/285 
https://www.mintscan.io/akash/proposals/300
